pyproject.toml

```diff
@@ -23,6 +14,15 @@ exclude = [
     ".pytype",
 ]
 
+[tool.ruff.format]
+skip-magic-trailing-comma = true
+exclude = [
+    # Exclude protobuf files because they have long line lengths
+    # Ideally, we could configure Ruff to allow longer line lengths
+    # for just these files, by adding a ruff.toml config file at their root.
+    "**_pb2.pyi",
+]
```

Comment:
Ruff is so fast and fixers are already run on these files as part of the generator scripts, I feel like this isn't even necessary.
I just kept if for now to have an apples-to-apples comparison.